### PR TITLE
Only load inline-edit-post when not viewing a single item

### PR DESCRIPTION
**Pull request by cklosowski**
#4015

### CODE REVIEW

File: includes/scripts.php [hunk 1]
```diff
@@ -171,7 +171,19 @@ function edd_load_admin_scripts( $hook ) {
 	wp_register_script( 'jquery-chosen', $js_dir . 'chosen.jquery' . $suffix . '.js', array( 'jquery' ), EDD_VERSION );
 	wp_enqueue_script( 'jquery-chosen' );
 
-	wp_register_script( 'edd-admin-scripts', $js_dir . 'admin-scripts' . $suffix . '.js', array( 'jquery', 'inline-edit-post' ), EDD_VERSION, false );
+	$admin_deps = array();
```
Comment (pippinsplugins) [inline includes/scripts.php]: @cklosowski This should use our `edd_is_admin_page()` function.

**Comment by cklosowski** [comment]
@pippinsplugins updated.

File: includes/scripts.php (hunk 1 above)
Comment (cklosowski) [inline includes/scripts.php]: See below, the update

Comment (pippinsplugins) [inline includes/scripts.php]: Looks good!

**Pull request merged**